details.js

```diff
@@ -1,204 +1,763 @@
+	model[in2]=new ScilabDouble(new data(1,0,0));
+	model[intyp]=new ScilabDouble(new data(-1,0,0));
+	model[evtin]=new ScilabDouble(new data(-1,0,0));
+	model[evtout] = new ScilabDouble(new data(-1, 0, 0), new data(-1, 0, 1)) // 1, 1 -> -1, -1 inverse
+	model[blocktype] = new ScilabString(new data('l', 0, 0));
+	model[firing] = new ScilabDouble(new data(-1, 0, 0), new data(-1, 0, 1)); // inverse
+	model[dep_ut] = new ScilabBoolean(new data('true', 0, 0), new data('false', 1, 0));
+	model[nmode] = new ScilabDouble(new data(1,0,0));
+	model[nzcross] = new ScilabDouble(new data(1,0,0));
+	
+	var gr_i = new ScilabString(new data("xstringb(orig(1),orig(2),\"IFTHEL_f\",sz(1),sz(2));", 0, 0));
+	var exprs=new ScilabString(new data("1",0,0),new data("1",0,1)); //value model.in, model.nmode inverse
+	var block = new standard_define(new ScilabDouble(new data(80, 0, 0), new data(80, 1, 0)), model, exprs, gr_i); // 3 -> 80
+	block[graphics][style] = new ScilabString(new data("IFTHEL_f", 0, 0));
+	return block;
+}
+
+
+
+function CLKSPLIT_f() {
+	var model = scicos_model();
+	model[sim] = new ScilabString(new data('split', 0, 0));
+	model[evtin]=new ScilabDouble(new data(-1,0,0));
+	model[evtout] = new ScilabDouble(new data(-1, 0, 0), new data(-1, 0, 1)) // 1, 1 -> -1, -1 inverse
+	model[blocktype] = new ScilabString(new data('d', 0, 0));
+	model[firing] = new ScilabDouble(new data(-1, 0, 0), new data(-1, 0, 1)); // inverse
+	model[dep_ut] = new ScilabBoolean(new data('true', 0, 0), new data('false', 1, 0));
+	
+	var gr_i = new ScilabString(new data("xstringb(orig(1),orig(2),\"CLKSPLIT_f\",sz(1),sz(2));", 0, 0));
+	
+	var block = new standard_define(new ScilabDouble(new data(80, 0, 0), new data(80, 1, 0)), model, new ScilabDouble(), gr_i); // 1 -> 80
+	block[graphics][style] = new ScilabString(new data("CLKSPLIT_f", 0, 0));
+	return block;
+}
+
+
+function ANDLOG_f() {
+	var model = scicos_model();
+	model[sim] = new ScilabString(new data('andlog', 0, 0));
+	model[out] = new ScilabDouble(new data(1, 0, 0));
+	model[out2] = new ScilabDouble(new data(1, 0, 0)); // null -> 1
+	model[evtin] = new ScilabDouble(new data(-1, 0, 0), new data(-1, 1, 0)) // 1, 1 -> -1, -1
+	model[blocktype] = new ScilabString(new data('d', 0, 0));
+	model[firing] = new ScilabDouble();
+	model[dep_ut] = new ScilabBoolean(new data('false', 0, 0), new data('false', 1, 0));
+
+	var gr_i = new ScilabString(new data("xstringb(orig(1),orig(2),\"ANDLOG_f\",sz(1),sz(2));", 0, 0));
+	var block = new standard_define(new ScilabDouble(new data(80, 0, 0), new data(80, 1, 0)), model, new ScilabDouble(), gr_i); // 3 -> 80
+	
+	// Style
+	block[graphics][out_implicit] = new ScilabString(new data("E",0,0));
+	//block.graphics.out_style = "ExplicitOutputPort;align=right;verticalAlign=middle;spacing=10.0;rotation=0";
+	block[graphics][style] = new ScilabString(new data("ANDLOG_f", 0, 0));
+	return block;
+}
+
+function BasicBlock()
+{
+	switch(arguments[0])
+	{
+		case "ANDBLK":
+			var attributes = {
```

Comment:
Make a function for attributes - similar to how we are doing in all other cases!